src/shell/dotnet/Shell/App.xaml.cs

```diff
@@ -170,5 +172,16 @@ private void InjectMessageRouterConfig()
                     
                 """);
     }
+   
+    private void InjectFDC3()
```

Comment:
I don't think FDC3 should be injected by default. It should be an opt-in feature that you can enable in `WebWindowOptions`, preferably in a `Features` collection so that it can be factored out into a plugin later (without breaking existing apps).

Reply:
I agree, but let's do that on a separate PR.